### PR TITLE
chore(typings): rename protractor to browser and add a protractor namespace

### PR DESCRIPTION
**Pull request by cnishina**
rename protractor to browser and add a protractor namespace

* added wrapDriver method from the browser.ts and ExpectedConditions
* added a new instance of ExpectedConditions to the protractor namespace
* imported selenium webdriver ActionSequence, Key, Command, and CommandName to the protractor namespace
* added promise object to the browser class

### CODE REVIEW

**Comment by juliemr** [comment]
We spoke briefly, but let's split the change to `Browser` into a separate PR.
I decided I'm fine calling it just `Browser`, it's clear in context and won't namespace conflict (yay typescript!)

File: lib/browser.ts [hunk 1]
```diff
@@ -184,7 +185,7 @@ export class Protractor {
    *
    * @type {Plugins} Object containing plugin funtions from config.
    */
-  private plugins_: Plugins;
+  plugins_: Plugins;
```
Comment (mgiambalvo) [inline lib/browser.ts]: Why are these no longer private?

Comment (cnishina) [inline lib/browser.ts]: They were marked as private; however, [lib/browser.ts](https://github.com/angular/protractor/blob/9b48143f90077b2d02f70d0dff19c376bb4c6824/lib/runner.ts#L199) makes a call to a private variable. This is the same issue for all the private variables in browser. They probably should be using a Get method.

Comment (mgiambalvo) [inline lib/browser.ts]: Maybe double check with Julie, but if they're not actually private then we should just drop the underscore at the end and remove the @private annotations in the jsdoc as well.

Comment (juliemr) [inline lib/browser.ts]: Hm yeah, these were fake private. We don't really want them as part of our documented API really, but the runner needs to access them. I'm fine making them public for now.

File: spec/basic/elements_spec.js [hunk 2]
```diff
@@ -155,7 +155,7 @@ describe('ElementFinder', function() {
 
     var invalidElement = element(by.binding('INVALID'));
     invalidElement.getText().then(function(value) {
-      successful.fulfill(true);
+      success.fulfill(true);
```
Comment (mgiambalvo) [inline spec/basic/elements_spec.js]: Is this right? It looks like the promise is "success" everywhere else.

Comment (mgiambalvo) [inline spec/basic/elements_spec.js]: It looks like the promise is "succesful" everywhere else? Is this right?

**Pull request merged**